pytorch_lightning/strategies/launchers/subprocess_script_hydra.py

```diff
@@ -0,0 +1,82 @@
+import os
+from typing import List, Optional, Tuple
+
+import __main__
+import torch
+
+from pytorch_lightning.strategies.launchers.subprocess_script import _SubprocessScriptLauncher
+from pytorch_lightning.utilities import _HYDRA_AVAILABLE
+
+if _HYDRA_AVAILABLE:
+    from hydra.core.hydra_config import HydraConfig
+    from hydra.utils import to_absolute_path
+
+
+class _SubprocessScriptHydraLauncher(_SubprocessScriptLauncher):
+    """Hydra Launcher subclasses `_SubprocessScriptLauncher to manage Hydra specific commands when running under
+    Hydra environment."""
```

Comment:
Simpler description, it seems redundant, and I wouldn't mention the subclass necessarily:
```suggestion
    """Hydra Launcher to support Hydra commands."""
```